app/templates/pdf/ticket_attendee.html

```diff
@@ -1,96 +1,182 @@
 <!--suppress ALL -->
 <html xmlns:pdf="https://github.com/xhtml2pdf/xhtml2pdf/blob/master/doc/usage.rst">
+
 <head>
-    <meta http-equiv="content-type" content="text/html; charset=utf-8"/>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
     <style>
-        @page {
```

Comment:
this should not be removed, also check whether these css tags are supported or not?